modules/caddyhttp/requestbody/requestbody.go

```diff
@@ -42,6 +44,10 @@ func (RequestBody) CaddyModule() caddy.ModuleInfo {
 }
 
 func (rb RequestBody) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhttp.Handler) error {
+	if rb.Replace != "" {
+		r.Body = io.NopCloser(strings.NewReader(rb.Replace))
```

Comment:
If we're completely replacing the request body, we miiiight want to call `r.Body.Close()` before re-assigning a new body. Yes the std lib will close it for us when the request is over, but would closing it early (and releasing those resources) cause any problems?

Also, I wonder if we should add placeholder support and expand those when doing the body replacement.

Reply:
I just don't know how placeholder are working right now, will search.